charter/README.md

```diff
@@ -102,6 +102,16 @@ Chairs have responsibilities to their parent Working Group: they ensure that the
 
 Chairs have responsibility to all participants in transparently reporting the group's activity.
 
+Chairs also have the following regular duties within their Working Groups:
+* Collect agenda items prior to each meeting to ensure a smooth and focused Working Group meeting.
+* Notify the Working Group channel at least 12 hours before the meeting, outlining the agenda.
+* Attend and lead each Working Group meeting.
+    * If the Chair is unable to attend a given meeting, they should delegate a pre-appointed person to lead the meeting in their stead.
+* Ensure meeting notes are uploaded to the `governance` repo in the Working Group's respective sub-folder following each meeting.
+* Check into persistent issues present on a given Working Group's repositories of responsibility (ex. bot failures)
+
```

Comment:
Does it make sense to add a bullet for something about action items from meetings:

> Ensure WG `Action Items` are actively being worked on

Reply:
i think every working group does action items differently, so i would prefer not to be prescriptive about this i think!